lattices/src/lib.rs

```diff
@@ -3,14 +3,24 @@
 
 use std::cmp::Ordering::{self, *};
 
-pub use cc_traits;
 use sealed::sealed;
+pub use {cc_traits, variadics};
 
 /// Module for definiting algebraic structures and properties.
 pub mod algebra;
 pub mod collections;
 mod conflict;
 mod dom_pair;
+/// GHT from the Wang/Willsey/Suciu Freejoin work
+pub mod ght;
+/// Lattice methods for GHT
+pub mod ght_lattice;
+/// COLT from the Wang/Willsey/Suciu Freejoin work
+pub mod ght_lazy;
+/// Macros for GHT
+pub mod ght_macros;
+/// Tests for GHT
+pub mod ght_test;
```

Comment:
nit: Would be good to move these into their respective files actually, so `//! GHT from the Wang/Willsey/Suciu Freejoin work` at the top of `ght.rs` instead of here